docs/HOW_TO_UPGRADE.md

```diff
@@ -0,0 +1,148 @@
+      * "app/views/form/decidim/participatory_procceses/admin/participatory_procces_form.rb"
+      * "app/queries/decidim/participatory_processes/not_regulation.rb"
+      * "app/queries/decidim/participatory_processes/organization_published_participatory_processes.rb"
+      * "app/views/decidim/participatory_procceses/admin/participatory_processes/form.html.erb"
+      * "app/views/decidim/participatory_processes/pages/home/highlihted_processes.html.erb"
+      * "config/locales/"
+
+
+  4. Decidim Search User ("decidim-admin")
+      This module add a search user in admin site.
+
+      The files modified are:
+      * "app/controllers/decidim/admin/users_controller.rb"
+      * "app/views/decidim/admin/users/index.html.erb"
+      * "config/locales/"
+
+
+  5. Decidim Selectable News ("decidim-core")
+      This module adds a functionallity to specify the participatory process followers to send the newsletter
+
+      The files modified are:
+      * "app/commands/decidim/admin/deliver_newsletter.rb"
+      * "app/controllers/decidim/admin/newsletters_controller.rb"
+      * "app/jobs/decidim/admin/newsletter_job.rb"
+      * "app/views/decidim/admin/newsletters/index.html.erb"
+      * "app/views/decidim/admin/newsletters/show.html.erb"
+      * "app/views/decidim/admin/newsletters/processes_div.html.erb"
+      * "config/locales/"
+
+
+  6. Also there, are custom files in the participa.
+
+      The files modified are:
+      * "app/assets/fonts/"
+      * "app/assets/stylesheets/"
+      * "app/controllers/decidim/proposals/proposals_controller.rb"
+      * "app/controllers/decidim_controller.rb"
+      * "app/helpers/decidim/participatory_processes/admin/"
+      * "app/views/decidim/assemblies/assemblies/index.html.erb"
+      * "app/views/decidim/assemblies/order_by_assemblies.html.erb"
+      * "app/views/decidim/pages/index.html.erb"
+      * "app/views/decidim/proposals/proposals/index.html.erb"
+      * "app/views/decidim/proposals/proposals/index.js.erb"
+      * "app/views/decidim/proposals/proposals/show.html.erb"
+      * "app/views/decidim/proposals/proposals/most_voted_comments.html.erb"
+      * "app/views/layouts/decidim/mailer.html.erb"
+      * "config/locales/"
+
+### New modules
+  1. Decidim Admin Extended ("decidim-admin"):
+  add the necessary layouts to avoid breaking future features of Decidim with the applied customizations of other modules, like Department, Theme or Type.
+
+  2. Decidim Department ("decidim-department"): Add a CRUD engine to create new Departments
+
+  3. Decidim Theme ("decidim-theme"): Add a CRUD engine to create new Theme
+
+  4. Decidim Type ("decidim-type"): Add a CRUD engine to create new Type
+
+  5. Decidim Regulations ("decidim-regulations"): Group the ParticipatoryProcesses with a ParticipatoryProcessGroup and showed in a clone of index iew of ParticipatoryProcesses called Regulations.
```

Comment:
s/Group/Groups/

Can you please rewrite the sentence, in the middle it gets difficult to understand